src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -550,6 +551,138 @@ def test_views_lti_video_student_live_mode_on(
         # signature)
         self.assertEqual(mock_verify.call_count, 1)
 
+    @mock.patch.object(LTI, "verify")
+    @mock.patch.object(LTI, "get_consumer_site")
+    @override_settings(SENTRY_DSN="https://sentry.dsn")
+    @override_settings(RELEASE="1.2.3")
+    @override_settings(JITSI_ENABLED=True)
+    def test_views_lti_video_student_live_mode_on_with_jitsi(
+        self, mock_get_consumer_site, mock_verify
+    ):
+        """Validate the format of the response for a live video requested by a student."""
```

Comment:
I would add the context in which you added this test
```suggestion
        """
        Validate the format of the response for a live video requested by a student.
        The student should see a restricted version of the live info to allow a connection to jitsi
        """
```

Reply:
this test does not exists anymore